pkg/userdata/flatcar/testdata/v1.17.0.yaml

```diff
@@ -1,464 +1 @@
er.target\n","dropins":[{"contents":"[Unit]\nRequires=download-script.service\nAfter=download-script.service\n","name":"40-docker.conf"}],"enabled":true,"name":"kubelet-healthcheck.service"},{"contents":"[Unit]\nDescription=Setup Kubelet Node IP Env\nRequires=network-online.target\nAfter=network-online.target\n\n[Service]\nExecStart=/opt/bin/setup_net_env.sh\nRemainAfterExit=yes\nType=oneshot\n[Install]\nWantedBy=multi-user.target\n","enabled":true,"name":"nodeip.service"},{"contents":"[Unit]\nDescription=Kubernetes Kubelet\nRequires=docker.service\nAfter=docker.service\n[Service]\nTimeoutStartSec=5min\nCPUAccounting=true\nMemoryAccounting=true\nEnvironmentFile=-/etc/environment\nEnvironmentFile=/etc/kubernetes/nodeip.conf\nEnvironment=PATH=/bin:/sbin:/usr/bin:/usr/sbin:/usr/local/bin:/usr/local/sbin:/opt/bin\nExecStartPre=/bin/bash /opt/bin/setup_net_env.sh\nExecStartPre=/bin/mkdir -p /var/lib/calico\nExecStartPre=/bin/mkdir -p /etc/kubernetes/manifests\nExecStartPre=/bin/mkdir -p /etc/cni/net.d\nExecStartPre=/bin/mkdir -p /opt/cni/bin\nExecStartPre=/bin/bash /opt/load-kernel-modules.sh\nExecStartPre=/bin/sh -c '/usr/bin/env \u003e /tmp/environment'\nExecStart=/usr/bin/docker run --name %n \\\n  --rm --tty --restart no \\\n  --network host \\\n  --pid host \\\n  --env-file /tmp/environment \\\n  --privileged \\\n  --cgroup-parent system.slice \\\n  --entrypoint kubelet \\\n  -v /dev:/dev \\\n  -v /etc/cni/net.d:/etc/cni/net.d \\\n  -v /etc/kubernetes:/etc/kubernetes \\\n  -v /etc/machine-id:/etc/machine-id:ro \\\n  -v /etc/os-release:/etc/os-release:ro \\\n  -v /etc/resolv.conf:/etc/resolv.conf:ro \\\n  -v /lib/modules:/lib/modules \\\n  -v /mnt:/mnt:rshared \\\n  -v /opt/cni/bin:/opt/cni/bin:ro \\\n  -v /run:/run \\\n  -v /sys:/sys \\\n  -v /usr/sbin/iscsiadm:/usr/sbin/iscsiadm \\\n  -v /var/lib/calico:/var/lib/calico:ro \\\n  -v /var/lib/cni:/var/lib/cni \\\n  -v /var/lib/docker:/var/lib/docker \\\n  -v /var/lib/kubelet:/var/lib/kubelet:rshared \\\n  -v /var/log/pods:/var/log/pods \\\n  for-kubernetes-less-then-1.19/hyperkubeImage:v1.17.0 \\\n  --bootstrap-kubeconfig=/etc/kubernetes/bootstrap-kubelet.conf \\\n  --kubeconfig=/var/lib/kubelet/kubeconfig \\\n  --config=/etc/kubernetes/kubelet.conf \\\n  --network-plugin=cni \\\n  --cni-conf-dir=/etc/cni/net.d \\\n  --cni-bin-dir=/opt/cni/bin \\\n  --cert-dir=/etc/kubernetes/pki \\\n  --cloud-provider=vsphere \\\n  --cloud-config=/etc/kubernetes/cloud-config \\\n  --hostname-override=node1 \\\n  --dynamic-config-dir=/etc/kubernetes/dynamic-config-dir \\\n  --exit-on-lock-contention \\\n  --lock-file=/tmp/kubelet.lock \\\n  --volume-plugin-dir=/var/lib/kubelet/volumeplugins \\\n  --node-ip ${KUBELET_NODE_IP}\nExecStop=-/usr/bin/docker stop %n\nRestart=always\nRestartSec=10\n[Install]\nWantedBy=multi-user.target\n","enabled":true,"name":"kubelet.service"},{"dropins":[{"contents":"[Service]\nEnvironmentFile=-/etc/environment\n","name":"10-environment.conf"}],"enabled":true,"name":"docker.service"}]}}
```

Comment:
I will change that to more readable form. For now i just wanted to see that unit tests are passing.